backend/app/services/execution/schedule_helper.py

```
@@ -227,6 +250,46 @@ async def _dispatch_task_async(task_id: int) -> None:
         db.close()
 
 
+async def _recover_executor(
+    db: Session,
+    subtask: "Subtask",
```

Ruff (0.15.6)

[error] Undefined name `Subtask`

(F821)